activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+*   Allow loading database dumps from stdin with `-`.
+
```

Comment:
Can you fix the conflict in the CHANGELOG and also add your name here. Like this:

```
*   Allow loading database dumps from stdin with `-`.

    *Michal Papis*
```